enhancements/network/ovs-hardware-offload.md

```diff
@@ -0,0 +1,242 @@
+---
+title: ovs-hardware-offload
+authors:
+  - "@zshi"
+reviewers:
+  - "@dcbw"
+  - "@squeed"
+  - "@danwinship"
+  - "@pliurh"
+  - "@dougbtv"
+  - "@s1061123"
+approvers:
+  - "@knobunc"
+  - "@fepan"
+creation-date: 2020-04-27
+last-updated: 2020-04-27
+status: implementable
+see-also: []
+replaces: []
+superseded-by: []
+
+---
+
+# OVS-Hardware-Offload
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+- When will RHCOS uses RHEL-8.3 as base image?
+- Any known issues with connction track feature using OVS Hardware Offload?
+- Any known issues with IPv6 traffic using OVS Hardware Offload?
+
+## Summary
+
+This enhancement is to introduce OVS hardware offload feature in OpenShift
+for high-performance datapath with OVN-Kubernetes. OVS hardware offload
+enables offloading of OVS datapath rules to a compatible network card using
+linux Traffic Control (TC) tool `tc`. It relies on SR-IOV technology to control
+the SR-IOV Virtual Functions (VF) via VF representor port that is plugged
+into OpenvSwitch bridge. It improves the utilization of available bandwidth
+on underlying NICs by moving CPU intensive network packet processing from
+OVS software to hardware.
+
+## Motivation
+
+Enable high-performance datapath on primary pod interface with production-ready
+OVN-Kubernetes.
+
+### Goals
+
+Enable OpenFlow hardware offload for OVN datapath
+
+### Non-Goals
```

Comment:
- Using hardware offload for any part of the OpenShift control plane

Reply:
updated